workspaces/github-actions/plugins/github-actions/README.md

```diff
@@ -71,6 +71,11 @@ TBD
 yarn --cwd packages/app add @backstage-community/plugin-github-actions
 ```
 
+Now, add this dependency to your backend index file (new backend system):
```

Comment:
Before this line we should note that this will already be in place if you are using GitHub Auth to sign in but if your don't do this. We should also include the command to add this package. Maybe something like this:

> If you aren't using GitHub Auth to sign into your Backstage instance you will need to add the following:
> 
> ```bash
> yarn --cwd packages/backend add @backstage/plugin-auth-backend-module-github-provider
> ```

Then you content would be next. Does that make sense?

Reply:
It does make total sense, I'll change it!

Reply:
This is part way there, they may not have that package added either so we need to make use to include the command for that or they will have errors in the backend `index.ts` 👍

Reply:
Done, I added the missing package installation to the readme!